sunbeam-python/sunbeam/plugins/dns/README.md

```diff
@@ -0,0 +1,46 @@
+# dns service
+
+This plugin provides a DNS service for Sunbeam. It based on [Designate](https://docs.openstack.org/designate/latest/), a DNS service for OpenStack and Bind.
```

Comment:
I think there are multiple DNS backends possible, and we have chosen Bind?

Reply:
Yes, current deployment is only Bind